ddtrace/contrib/langchain/patch.py

```diff
@@ -144,6 +145,22 @@ def _tag_openai_token_usage(
         _tag_openai_token_usage(span._parent, llm_output, propagated_cost=propagated_cost + total_cost, propagate=True)
 
 
+def _is_workflow(instance):
+    if LLMObs._bedrock_enabled() and _is_bedrock_llm_instance(instance):
+        return True
+    if LLMObs._openai_enabled() and _is_openai_chat_instance(instance) or _is_openai_llm_instance(instance):
+        return True
+    return False
+
+
+def _is_bedrock_llm_instance(instance):
+    """Safely check if a traced instance is a Bedrock LLM."""
+    try:
+        return isinstance(instance, BASE_LANGCHAIN_MODULE.llms.Bedrock)
+    except (AttributeError, ModuleNotFoundError, ImportError):
+        return False
+
+
```

Comment:
Can we move this checking to `ddtrace.llmobs.integrations.langchain`? I'd like to avoid polluting integration patch file if possible.

Reply:
Also if we can avoid checking `isinstance(instance, ...)` directly would be great, we can either 1) check `instance._llm_type` or 2) check the span's `model_provider` tag instead